spec/models/lead_provider_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe LeadProvider, type: :model do
+  it "can be created" do
+    expect {
+      LeadProvider.create(name: "Test Lead Provider")
```

Comment:
I kind of feel this is testing rails itself. Not sure it's strictly necessary but it does no harm.